### PR TITLE
Improve article teaser UX

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard10**
## Summary

- The full teaser is now clickable
- On hover the teaser image will zoom in a little bit
- On hover the teaser title will tween to blue
- Created a StyledComponent-instance of the `Heading`-component, this was necessary in order to target these components as selector.

### CODE REVIEW

**Review by VWSCoronaDashboard4 — APPROVED** [review]

File: packages/app/src/components-styled/article-teaser.tsx [hunk 1]
```diff
@@ -25,37 +26,66 @@ export function ArticleTeaser(props: ArticleLinkProps) {
...
+            {siteText.common.read_more}
+            <Arrow />
+          </InlineText>
+        </Box>
+      </StyledArticleTeaser>
+    </Link>
+  );
+}
+
+const StyledArticleTeaser = styled.a(
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/components-styled/article-teaser.tsx:46]: nit; I think it's a little odd to have a StyledArticleTeaser as a child of ArticleTeaser. Maybe give it a more generic name like `Container`

File: packages/app/src/components-styled/typography/heading.tsx [hunk 2]
```diff
@@ -26,61 +26,27 @@ export type HeadingLevel = 1 | 2 | 3 | 4 | 5;
...
   TypographyProps &
   ColorProps;
 
-const composedStyles = [
-  css({
-    fontFamily: 'body',
-    lineHeight: 2,
-    mt: 0,
-  }),
+const levelStyles: Record<HeadingLevel, ReturnType<typeof css>> = {
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/components-styled/typography/heading.tsx:35]: I need to start using this `ReturnType<typeof` thing. It's not in my system yet :)

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: packages/app/src/components-styled/article-teaser.tsx (hunk 1 above)
Comment (VWSCoronaDashboard10) [reply packages/app/src/components-styled/article-teaser.tsx:46]: Let's discuss this nitpick during the next frontend meeting ;-)

File: packages/app/src/components-styled/typography/heading.tsx (hunk 2 above)
Comment (VWSCoronaDashboard10) [reply packages/app/src/components-styled/typography/heading.tsx:35]: yes it's a nice invention for the lazy among us

**Pull request merged**